docs/channel/notificatins-from-notification-backend.md

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
there's a typo in the file's name `notificatins` instead of `notifications`

Reply:
Fixed